### PR TITLE
Some small changes

### PR DESCRIPTION
**Pull request by Makkkkus**
* Made the color of the active item text white.
* Standardized code; makes it look better.
* Fixed the Norwegian localization. Added, "ÆØÅ.
* Fixed a spelling mistake.
* Made a parent class for all the wool tiles, which makes it easier to expand behavior.

### CODE REVIEW

File: src/minicraft/item/Item.java [hunk 1]
```diff
@@ -29,17 +29,22 @@ protected Item(String name, Sprite sprite) {
...
-		sprite.render(screen, x, y);
-		if(ininv) {
+	// TODO this method (and Menu.renderItemList) is actually slowly getting deprecated; I just haven't gotten around to updating all the menus yet.
+	public void renderInventory(Screen screen, int x, int y, boolean ininv) { 
+		if (ininv) {
+			String dispName = getDisplayName();
+			sprite.render(screen, x, y);
 			String shortname = dispName.length() > 20 ? dispName.substring(0, 20) : dispName;
 			Font.draw(shortname, screen, x + 8, y, Color.WHITE);
+		} else
```
Comment (africalinguist) [inline src/minicraft/item/Item.java:39]: is it just me or is there a missing bracket here?

File: src/minicraft/level/tile/wool/Wool.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import minicraft.item.Item;
+import minicraft.item.Items;
+import minicraft.item.ToolItem;
+import minicraft.item.ToolType;
+import minicraft.level.Level;
+import minicraft.level.tile.Tile;
+import minicraft.level.tile.Tiles;
+
+public class Wool extends Tile {
+	private String woolType;
```
Comment (africalinguist) [inline src/minicraft/level/tile/wool/Wool.java:17]: odd indentation here

File: src/minicraft/item/Item.java (hunk 1 above)
Comment (Makkkkus) [reply src/minicraft/item/Item.java:39]: No, that's the end of the function.

**Comment by Makkkkus** [comment]
Fixed

**Comment by africalinguist** [comment]
all right

**Pull request merged**